src/pages/organisationUnits/list/OrganisationUnitList.tsx

```diff
@@ -105,18 +105,19 @@ export const OrganisationUnitList = () => {
         (column) => column.meta.fieldFilter
     )
 
+    const { queries, fetchNextPage } = usePaginatedChildrenOrgUnitsController({
+        parentIds: parentIdsToLoad,
+        fieldFilters,
+    })
     const isFiltering = !!identifiableFilter
+    const hasErrored = queries.find((query) => query.error) !== undefined
```

Comment:
Very minor, almost don't want to comment on this - but this would be a bit more dogmatic. 
```suggestion
    const hasErrored = queries.some((query) => query.isError)
```